core/export.ts

```diff
@@ -0,0 +1,158 @@
+import { loadValue } from '@logux/client'
+import { atom, onMount } from 'nanostores'
+
+import {
+  type CategoryValue,
+  feedsByCategory,
+  getCategories,
+  getCategoryTitle
+} from './category.js'
+import { type FeedValue, getFeeds } from './feed.js'
+import { readonlyExport } from './lib/stores.js'
+import { getPosts, type PostValue } from './post.js'
+
+type ExtendedFeedValue = {
```

Comment:
`FeedWithPosts` will be better, since `Extended` doesn’t explain how exactly it is extentended.